cli.js

```diff
@@ -58,6 +61,34 @@ function log(report) {
 	process.exit(report.errorCount === 0 ? 0 : 1);
 }
 
+function open(report) {
+	if (report.errorCount === 0) {
+		return;
+	}
+
+	var files = report.results
+		.filter(function (file) {
+			return file.errorCount > 0;
+		})
+		.map(function (file) {
+			return file.filePath;
+		});
+
+	var editor = process.env.EDITOR;
+
+	if (!editor) {
+		console.log();
+		console.log('`open` option was used, but your $EDITOR environment variable is empty.');
+		console.log('Fix it by setting path to your editor of choice in ~/.bashrc or ~/.zshrc:');
+		console.log();
+		console.log('    export EDITOR=atom');
+		console.log();
+		return;
+	}
+
```

Comment:
Webstorm supports it as well. Default command is `wstorm`, I usually use `ws` though.

<img width="760" alt="screenshot 2015-12-13 17 40 38" src="https://cloud.githubusercontent.com/assets/4082216/11769934/ade1d2a4-a1c0-11e5-9af8-8b497e23c1c0.png">